src/main/java/uk/gov/hmcts/reform/mi/miextractionservice/domain/SourceProperties.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.mi.miextractionservice.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Data
+public class SourceProperties {
```

Comment:
Opinion: use 
@Value
@Builder
@ConstructorBinding
It gives you immutable model and allow spring to load the property using all arguments constructor